unabridged-works-volume-1/original/06-jew-outward.adoc

```diff
@@ -871,9 +906,13 @@ who "`trod under foot the holy city.`" Rev. 11:2. And we have found by experienc
 that, while we ourselves were crying up the outward court,
 we did trample under foot the city, though we then knew it not.
 
+====
+
+[.discourse-part]
```

Comment:
So I wasn't sure what to do with these because they don't have a number or _Question_ or anything, but it seems to need something because of the _Answer_ below it.

Reply:
I think you did the right thing. It feels like a discourse, and the answers show that there is a back-and-forth type thing, so this feels good to me. 👍